charts/vault/Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash # Use bash syntax
+ROOT_DIR:=../..
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/hack/make-rules/docker.mk
+
+KUBE_NAMESPACE ?= m4d-system
+WITHOUT_OPENSHIFT ?= true
```

Comment:
Consider changing to proactive OPENSHIFT flag defaulted to false and just always set its value to .global.openshift

Reply:
Sure will do that. tnx